ui/webui/test/README.rst

```diff
@@ -243,3 +266,218 @@ development environment that's independent from the host::
 
     toolbox create --image quay.io/cockpit/tasks -c cockpit
     toolbox enter cockpit
+
+Web UI integration tests
+========================
+
+Integration (or end to end) tests are stored in `./ui/webui/test/integration` 
+together with tplib test cases and test plans (those are only needed when running
+the tests using Permian).
+
+Integration tests use one more level of abstraction, to make it easier to write new
+tests and avoid code duplication, class `IntegrationTest <https://github.com/rhinstaller/anaconda/blob/master/ui/webui/test/helpers/integration.py#L38>`_.
+This class handles the process of going through all the required installation steps
+with default options. So when writing new test you only have to use this class as
+parent and reimplement the functions that are important for the test case.
+
+Integration tests examples
```

Comment:
Good catch, they are actually already outdated. I think it would be better instead of making some example testplan I will transform them to real testcases that will be part of the periodic testing and use them as examples.

Reply:
New test cases added in #4729